packages/core/src/lib/uikitExcludePattern.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 
+var _ = require('lodash');
+
 const uikitExcludePattern = (pattern, uikit) => {
   const state = pattern.patternState;
-  return uikit.excludedPatternStates.includes(state);
+  const tags = pattern.tags;
```

Comment:
In markdown, you could add one single tag by just providing something like the following. 
```
---
tags: foo-bar
---
```

This will result in only having a string in `pattern.tags` and results in intersection returning false. My suggestion is to check if the pattern tags are already an array and if not the string will be included in an array.

```suggestion
  const tags = _.isArray(pattern.tags) ? pattern.tags : [pattern.tags];
```